engine/release-notes/24.0.md

```diff
@@ -19,6 +19,20 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](../api/version-history.md).
 
+## 24.0.4
+
+{% include release-date.html date="2023-07-07" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.4 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.4)
+- [moby/moby, 24.0.4 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.4)
+
+### Bug fixes and enhancements
+
+* Fix a panic during live-restore of containers with bind mounts. [moby/moby#45903](https://github.com/moby/moby/pull/45903)
```

Comment:
Do we need to mention that it's was a regression in 24.0.3?

Reply:
cc @dvdksn; I have no strong opinion on that myself.

Reply:
Let's do it, helps explain the sudden release :)

Reply:
How's something like:
```suggestion
* Fix a regression introduced by 24.0.3: a panic during live-restore of containers with bind mounts. [moby/moby#45903](https://github.com/moby/moby/pull/45903)
```

Reply:
Or:
```suggestion
* Fix a panic during live-restore of containers with bind mounts (regression introduced in 24.0.3). [moby/moby#45903](https://github.com/moby/moby/pull/45903)
```

Reply:
last one seems fine.

first one I'd have written as "Fix  a regression introduced in 24.0.3 that cause a panic ..."